js/client/modules/@arangodb/testsuites/java.js

```diff
@@ -94,7 +94,7 @@ function javaDriver (options) {
         '-Dgroups=api',
         '-Dtest.useProvidedDeployment=true',
         '-Dtest.arangodb.version='+ db._version(),
-        '-Dtest.arangodb.isEnterprise=' + isEnterprise()? 'true' : 'false',
+        `-Dtest.arangodb.isEnterprise=${isEnterprise()? 'true' : 'false'}`,
         '-Dtest.arangodb.hosts=' + this.instanceManager.url.replace(rx,''),
         '-Dtest.arangodb.authentication=root:',
         '-Dtest.arangodb.topology=' + topology,
```

Comment:
These parameters are not supported anymore, as we discussed you should change https://github.com/arangodb/arangodb-java-driver/blob/main/test-functional/src/test/resources/arangodb.properties

Furthermore, you can only test `test-functional`, therefore need to add: `-am -pl test-functional`.